client/odie/query/index.ts

```diff
@@ -8,16 +8,16 @@ function odieSendMessage( messages: Message[], context: Context, chat_id?: strin
 	return wpcom.req.post( {
 		path,
 		apiNamespace: 'wpcom/v2',
-		body: { messages, context, chat_id: chat_id },
+		body: { messages, context, chat_id },
 	} );
 }
 
-function odieSendCustomMessage( messages: Message[], context: Context, chat_id?: string | null ) {
-	const path = `/odie/send_custom_message`;
+function odieSendSupportMessage( message: Message, context: Context, chat_id?: string | null ) {
+	const path = `/odie/support/chat/respond`;
```

Comment:
Sadly this won't work on Atomic. You'd need something like [this](https://github.com/Automattic/wp-calypso/blob/trunk/packages/help-center/src/hooks/use-messaging-availability.ts#L24-L35).